### PR TITLE
Add example for ^mimeType and ^mediaType metadata selectors

### PR DESCRIPTION
**Pull request by jesica-fera**

### CODE REVIEW

File: modules/ROOT/pages/dataweave-formats-multipart.adoc [hunk 1]
```diff
@@ -475,6 +476,127 @@ parsed.parts mapObject ((value, key, index) -> {(index): key})
 }
 ----
 
+[[example5]]
+=== Example: Using ^mediaType and ^mimeType Metadata Selectors
+
+The `^mimeType` selector returns the MIME type of a value without parameters, for example, `application/json`. +
```
Comment (dukesphere) [inline modules/ROOT/pages/dataweave-formats-multipart.adoc:482]:
```suggestion
The `^mimeType` selector returns the MIME type of a value without parameters, for example, `application/json`. 

```

Comment (dukesphere) [reply modules/ROOT/pages/dataweave-formats-multipart.adoc:482]: I don't think we use the `+` to start a new line like that in our docs.

Another option might be to provide some sort of intro sentence and describe `^mimeType` and `^mediaType` below that sentence in separate bullets.

File: modules/ROOT/pages/dataweave-formats-multipart.adoc [hunk 2]
```diff
@@ -475,6 +476,127 @@ parsed.parts mapObject ((value, key, index) -> {(index): key})
...
+The `^mediaType` selector by contrast includes any other related parameters related to the MIME type, for example, `application/json;charset=UTF-16`.
```
Comment (dukesphere) [inline modules/ROOT/pages/dataweave-formats-multipart.adoc:483]:
```suggestion
The `^mediaType` selector, by contrast, provides the MIME type and any other parameters related to it, for example, `application/json;charset=UTF-16`.
```

File: modules/ROOT/pages/dataweave-formats-multipart.adoc [hunk 3]
```diff
@@ -475,6 +476,127 @@ parsed.parts mapObject ((value, key, index) -> {(index): key})
...
+ "name": "mulesoft",
+ "id": 1
+}
+----------------------------049709565842371278701691--
+----
+
+=== Source
+
+The DataWeave script transforms the multipart input into multipart output by defining `output multipart/form-data` MIME type. The multipart output data contains three parts:
+* `orderAck` +
```
Comment (dukesphere) [inline modules/ROOT/pages/dataweave-formats-multipart.adoc:515]:
```suggestion

* `orderAck` +
```

Comment (dukesphere) [reply modules/ROOT/pages/dataweave-formats-multipart.adoc:515]: need blank line above the bullets

File: modules/ROOT/pages/dataweave-formats-multipart.adoc [hunk 4]
```diff
@@ -475,6 +476,127 @@ parsed.parts mapObject ((value, key, index) -> {(index): key})
...
+Contains a text with an acknowledge of the receipt of an order. The Content-Type is `text/plain`.
```
Comment (dukesphere) [inline modules/ROOT/pages/dataweave-formats-multipart.adoc:516]:
```suggestion
Contains a text with an acknowledgement of the receipt of an order. The Content-Type is `text/plain`.
```

File: modules/ROOT/pages/dataweave-formats-multipart.adoc [hunk 5]
```diff
@@ -475,6 +476,127 @@ parsed.parts mapObject ((value, key, index) -> {(index): key})
...
+* `order` +
+Contains the order itself. The Content-Type is `payload.parts.order.content.^mimeType`. The ^mimeType` selector becomes useful to generate the Content-Type field.
+* `originalContent`
+Contains the original content which is multipart. The Content-Type is `payload.^mediaType`. The ^mediaType selector includes useful metadata such as the original boundary.
```
Comment (dukesphere) [inline modules/ROOT/pages/dataweave-formats-multipart.adoc:520]:
```suggestion
Contains the original content, which is multipart. The Content-Type is `payload.^mediaType`. The ^mediaType selector includes useful metadata such as the original boundary.
```

**Review by dukesphere — COMMENTED** [review]
a few suggestions

**Review by teofr — DISMISSED** [review]

**Review by dukesphere — APPROVED** [review]

**Pull request merged**